<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaximiumMinimumForm</class>
 <widget class="QWidget" name="MaximiumMinimumForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maximum-minimum distance method</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Stop when</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLineEdit" name="distance"/>
   </item>
   <item row="2" column="8">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="9">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
